src/defs/metrics.d.ts

```diff
@@ -42,6 +42,20 @@ declare module "tsdDef" {
     }
 
     /**
+     * Interface for classes which create <code>Metrics</code> instances. Clients
```

Comment:
Yeah just followed the surrounding style. A bit of investigation reveals that doing a `grunt gendocs` generates the doc website, and the `<code>` tags do indeed have the expected effect, like this taken from the `Counter` docs:
<img width="860" alt="screen shot 2016-09-23 at 11 03 09 am" src="https://cloud.githubusercontent.com/assets/2703760/18796331/9658922e-817d-11e6-96f2-b8729237f7ef.png">